src/services/handleSorting.js

```diff
@@ -29,12 +40,46 @@ const handleSorting = (sortType, page = '') => {
     case 'Name':
       return (a, b) => a.tagname.localeCompare(b.tagname);
     case 'Username':
-      return (a, b) => a.username.localeCompare(b.username);
+      return (a, b) => a.localeCompare(b.username);
```

Comment:
Please fix this, this is breaking, just revert and it should work

Reply:
It is breaking when Name button is clicked but the other buttons are working.